dockstore-client/src/test/java/io/dockstore/client/cli/LaunchTestIT.java

```diff
@@ -240,7 +241,7 @@ private void yamlAndJsonEntryCorrect(String entryType) {
         args.add("--json");
         args.add(jsonTestParameterFile.getAbsolutePath());
         exit.expectSystemExitWithStatus(CLIENT_ERROR);
-        exit.checkAssertionAfterwards(() -> Assert.assertTrue(systemErrRule.getLog().contains("Missing required flag")));
+        exit.checkAssertionAfterwards(() -> Assert.assertTrue(systemErrRule.getLog().contains("Specify a test parameter file with either --json or --yaml not both")));
```

Comment:
Use a constant string shared with the implementation to make it easier for the next developer that changes the error text. i.e. developer will more easily be able to see that the error message is referenced in a test